src/contrib/cluster/Akka.Cluster.Tools/Singleton/reference.conf

```diff
@@ -61,6 +61,12 @@ akka.cluster.singleton-proxy {
 
   # Interval at which the proxy will try to resolve the singleton instance.
   singleton-identification-interval = 1s
+  
+  # Should the singleton proxy log a warning if no singleton actor were found after a period of time
+  log-singleton-identification-failure = true
+  
+  # The period the proxy will wait until it logs a missing singleton warning, defaults to 5 minutes 
+  singleton-identification-failure-period = 5m
```

Comment:
I'd shorten this to something more like 30 seconds and I'd keep raising warnings every time the period repeats without success - that way it's really, really clear that something is wrong in the logs

Reply:
Ok, that would actually make the design a lot simpler, I just need to run a repeating timer whenever `_singleton` is null and fire a warning

Reply:
Done